docs/DeveloperGuide.md

```diff
@@ -274,66 +274,141 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing to mouse interactions
+* a sociable member of NUS School of Computing
+* either has reasonable experience in computing or in search of a contact who is
+* prefer desktop apps to keep track of their contacts
 * is reasonably comfortable using CLI apps
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: add superior convenience in managing contacts from NUS SoC
 
 
 ### User stories
 
-Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+**Priorities:** 
+- High (must have) - `* * *`
+- Medium (nice to have) - `* *`
+- Low (unlikely to have) - `*`
+
+**Users:** 
+- anUdder (refers to any type of user described below)
+- brUdders (refers to mentee users and/or those who wish to mentor as peers)
+- mUdders (refers to mentor users)
+
+| Priority | As a…​                                   | I want to…​                       | So that I can…​                                                         |
+|----------|-----------------------------------------|----------------------------------|------------------------------------------------------------------------|
+| `* * *`  | anUdder                                 | add a new contact                |                                                                        |
+| `* * *`  | anUdder                                 | edit an existing contact         | update the added contact with new information                          |
+| `* * *`  | lonely brUdder who seeks help           | schedule a meeting               | remind myself of future meetings with another contact                  |
+| `* * *`  | curious anUdder                         | view the details of a contact    | learn more about the contacts abilities                                |
+| `* * *`  | anUdder                                 | list all contacts                |                                                                        |
+| `* * *`  | anUdder                                 | list all meetings with a contact |                                                                        |
+| `* * *`  | anUdder                                 | delete a contact                 | remove contacts that I no longer need                                  |
+| `* * *`  | anUdder                                 | filter through my contacts       | locate details of persons without having to go through the entire list |
+| `* *`    | new anUdder                             | see usage instructions           | refer to instructions when I forget how to use the App                 |
```

Comment:
LGTM, just a nit: consider changing "see usage instructions" can be changed to "get help" since its easier to understand